core/object/make_virtuals.py

```diff
@@ -198,6 +200,37 @@ def run(target, source, env):
 #define GDVIRTUAL_TRACK(m_virtual, m_initialized)
 #endif
 
+// MSVC WORKAROUND START
+// FIXME The below helper functions are needed to work around an MSVC bug.
+// They should be removed (by modifying core/object/make_virtuals.py) once the bug ceases to be triggered.
```

Comment:
Is there anywhere we can report this bug to MSVC so that this "FIXME" doesn't end up being a forever thing?

Reply:
They have a [report channel](https://developercommunity.visualstudio.com/cpp/report) that would be appropriate. However, they're [asking for an MRP](https://learn.microsoft.com/en-us/cpp/overview/how-to-report-a-problem-with-the-visual-cpp-toolset?view=msvc-170#how-to-prepare-your-report), which I cannot provide since I don't have an MSVC toolchain (I don't use Windows). I can however open an Issue on the Godot Issue Tracker, with all the information i have on it, and perhaps someone motivated will show up to create an MRP and report to the MSVC team?

Reply:
I use MSVC, so I can help. You can also compile on godbolt (but the MRP has to be short, for sure)